charts/cheetah-application/values.schema.json

```diff
@@ -0,0 +1,380 @@
+          }
+        }
+      }
+    },
+    "podSecurityContext": {
+      "description": "Security context for the entire pod. @default -- (see [values.yaml](values.yaml))",
+      "type": "object",
+      "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.28.0/podsecuritycontext.json"
+    },
+    "securityContext": {
+      "description": "Security context for the primary container. @default -- (see [values.yaml](values.yaml))",
+      "type": "object",
+      "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.28.0/securitycontext.json"
+    },
+    "monitoring": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Whether to enable Prometheus scraping by creating a ServiceMonitor resource",
+          "type": "boolean"
+        },
+        "port": {
+          "description": "Which port to look for Prometheus metrics",
+          "type": "number"
+        },
+        "path": {
+          "description": "Which path to look for Prometheus metrics",
+          "type": "string"
+        }
+      }
+    },
+    "pdb": {
+      "type": "object",
+      "properties": {
+        "create": {
+          "description": "Whether to create a PodDisruptionBudget for ensuring that an application is always available",
+          "type": "boolean"
+        },
+        "labels": {
+          "description": "Extra labels for the PodDisruptionBudget",
+          "type": "object"
+        },
+        "annotations": {
+          "description": "Extra annotations for the PodDisruptionBudget",
+          "type": "object"
+        },
+        "minAvailable": {
+          "description": "How many pod replicas must always be available after eviction. Ignored if 0",
+          "type": "number"
+        },
+        "maxUnavailable": {
+          "description": "How many pod replicas are allowed to to be unavailable during eviction. Ignored if 0",
+          "type": "number"
+        }
+      }
+    },
+    "autoscaling": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Whether to enable horizontal pod autoscaling",
+          "type": "boolean"
+        },
+        "minReplicas": {
+          "description": "Minimum number of replicas",
+          "type": "number"
+        },
+        "maxReplicas": {
+          "description": "Maximum number of replicas",
+          "type": "number"
+        },
+        "targetCPUUtilizationPercentage": {
+          "description": "Target CPU requests percentage utilization. Ignored if 0",
+          "type": "number"
+        },
+        "targetMemoryUtilizationPercentage": {
+          "description": "Target RAM requests percentage utilization. Ignored if 0",
+          "type": "number"
+        }
+      }
+    },
+    "nodeSelector": {
+      "type": "object",
+      "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.28.0/nodeselector.json"
```

Comment:
This is not the correct node-selector format (I am unsure that this one is used for).
You can see the correct format here: https://github.com/yannh/kubernetes-json-schema/blob/master/v1.25.9/podspec.json#L143

Reply:
Got it